docs/developer-guides/addresses.md

```diff
@@ -36,4 +36,6 @@ The table below provides the locking script and prefix details for each address
 
 ## P2PKH address generation flow
 
+When hashing a public key to generate an address (e.g. a Pay-to-PubkeyHash (P2PKH) address), Decred uses blake256r14 instead of SHA-256. The full algorithm to create an address is blake256r14+ripemd160+base58, as illustated below.
```

Comment:
I think this is a conflict from #35. Can you rebase this PR please.

Reply:
Oops, sorry bout that. Rebased

Reply:
A typo has been introduced here, "illustated"

Reply:
Fixed